mindsdb/exceptions.py

```diff
@@ -0,0 +1,18 @@
+class TableNotFound(Exception):
```

Comment:
I think this file should be located in 
mindsdb/integrations/libs, near by api_handler.py

Reply:
Yea, the exceptions.py was way too common. 
Renamed it to api_handler_exceptions.py, and moved it to mindsdb\integrations\libs